go/libraries/doltcore/env/actions/reset.go

```diff
@@ -136,26 +136,31 @@ func ResetHardTables(ctx context.Context, dbData env.DbData, cSpecStr string, ro
 	return resetHardTables(ctx, dbData, cSpecStr, roots)
 }
 
-func ResetHard(ctx context.Context, dEnv *env.DoltEnv, cSpecStr string, roots doltdb.Roots) error {
+// ResetHard resets the working, staged, and head based on the given parameters.
```

Comment:
and head to the ones in the Roots provided.

Reply:
Updated documentation as suggested.